routes/contents.js

```diff
@@ -0,0 +1,21 @@
+const express = require("express");
+
+const router = express.Router();
+
+const upload = require("./controllers/upload.controller");
+const uploadToLocal = require("../services/uploadToLocal");
+const transferController = require("./controllers/transfer.controller");
+
+router.post(
+  "/contents/files",
+  uploadToLocal.fields([
+    { name: "mainVideoFile" },
+    { name: "subOneVideoFile" },
+    { name: "subTwoVideoFile" },
+  ]),
+  upload,
+);
+
+router.post("/contents/urls", transferController.transferMedia);
```

Comment:
위에서 라우터 경로 설정에 대해 드린 코멘트의 변경 적용사항에 따라 아래와 같이 contents 라우터 내 경로 설정도 아래와 같이 변경 부탁드립니다! 🙇‍♀️

```suggestion
router.post(
  "/files",
  uploadToLocal.fields([
    { name: "mainVideoFile" },
    { name: "subOneVideoFile" },
    { name: "subTwoVideoFile" },
  ]),
  upload,
);

router.post("/urls", transferController.transferMedia);
```